android/src/main/java/ie/gov/tracing/nearby/StateUpdatedWorker.java

```diff
@@ -121,8 +121,15 @@ public StateUpdatedWorker(
 
               int[] ad = exposureSummary.getAttenuationDurationsInMinutes();
 
-              double[] tw = doubleArrayFromString(SharedPrefs.getString("thresholdWeightings", context));
-              long timeThreshold = SharedPrefs.getLong("timeThreshold", context);
+              double[] tw;
+              long timeThreshold;
+              if (simulate) {
```

Comment:
This is a missing piece from the previous PR?

Reply:
Yes found a bug in simulate where it failed on Android if you have never loaded a config file